packages/dcos-log/build

```diff
@@ -40,12 +42,14 @@ Description=DC/OS Log service agent: DC/OS logging service
 [Service]
 PermissionsStartOnly=True
 EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=/opt/mesosphere/etc/dcos-log.env
+EnvironmentFile=-/opt/mesosphere/etc/dcos-log-extra.env
 Restart=always
 StartLimitInterval=0
 RestartSec=5
 User=dcos_log
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-log-agent
-ExecStart=/opt/mesosphere/bin/dcos-log -config /opt/mesosphere/etc/dcos-log-config.json
+ExecStart=/opt/mesosphere/bin/dcos-log -config $DCOS_LOG_CONFIG_PATH 
```

Comment:
${ }

Basic environment variable substitution is supported. Use "${FOO}" as part of a word, or as a word of its own, on the command line, in which case it will be replaced by the value of the environment variable including all whitespace it contains, resulting in a single argument. Use "$FOO" as a separate word on the command line, in which case it will be replaced by the value of the environment variable split at whitespace, resulting in zero or more arguments. For this type of expansion, quotes are respected when splitting into words, and afterwards removed.

https://www.freedesktop.org/software/systemd/man/systemd.service.html